data/src/main/java/com/bottlerocketstudios/brarchitecture/data/model/SnippetDto.kt

```diff
@@ -9,9 +9,12 @@ import java.time.ZonedDateTime
 @JsonClass(generateAdapter = true)
 @Parcelize
 data class SnippetDto(
-    @Json(name = "id") val id: String? = null,
-    @Json(name = "title") val title: String? = null,
-    @Json(name = "is_private") val isPrivate: Boolean? = null,
-    @Json(name = "owner") val owner: UserDto? = null,
-    @Json(name = "updated_on") val updated: ZonedDateTime? = null,
+    val id: String? = null,
```

Comment:
Question:  Were the Json annotations auto generated and/or code standard?  I usually only use those when JSON name doesn't follow Kotlin variable naming standards, there is a mispelled word, etc.  I have no issue with adding them back in for all variables, just curious.

Reply:
@BottleRocket-Colin I don't remember how the data classes were originally built but they didn't define the json name explicitly until https://github.com/BottleRocketStudios/Android-ArchitectureDemo/commit/ce77220f6cc84add85e7e52662628bff18e6d9c9

> Explicitly specify @Json(name = ... ) for all network request/response data classes
> 
> Doing so provides the following benefits:
> 
> 1. Supports correct (property) naming conventions in the app regardless of serialization key naming conventions.
> 2. Prevents future property name refactoring breaking serialization

I'd say add them back to prevent against future runtime failures when/if the Dto properties are renamed.